java/src/jmri/jmrit/display/layoutEditor/TrackSegment.java

```diff
@@ -1326,20 +1335,22 @@ public void setCentreY(double y) {
     public Point2D getCentre() {
         Point2D result = new Point2D.Double(centreX, centreY);
 
-        Point2D ep1 = result;
-        Object c1 = getConnect1();
-        if (c1 != null) {
-            ep1 = layoutEditor.getCoords(getConnect1(), getType1());
-        }
+        if (false) {
```

Comment:
The code in this if statement is dead code. It should be commented out or removed instead of encapsulated in an if statement that is never executed.

Reply:
I kind of wanted it to pass all its integration checks before I dead-code stripped this… because if it fails I may have to put it back.
But… whatever makes you happy… ;-)

Reply:
We have a [CI process that runs in Jenkins](http://jmri.tagadab.com/jenkins/job/Development/job/Findbugs/) that *does* flag dead code, but since there are too many existent "normal" priority flags, its not currently configured to cause Travis to fail.